### ISSUE TITLE
katar causes crash when rightclicking trees / leaves 

### ISSUE THREAD

**Issue by puffpants** [sinkillerj/ProjectE#1915]
**Tickets that do not conform to this template will be closed without comment**

Exact ProjectE version (do not say "latest", "latest on Curse", or similar):  ProjectE-1.12.2-PE1.4.1

Exact Forge version: forge-1.12.2-14.23.5.2838

Link to crash log (please use a paste site, do not attach the .txt or paste the log inline):  https://paste.dimdev.org/mikagetili.mccrash

Steps to reproduce:

1.right click tree log / leaves

What I expected to happen:

the katar removed all the trees

What happened instead:

recive the "ProjectE ran into a problem! press Ctrl + i for more info"

**Comment by sinkillerj** [comment]
Does this only happen with a particular type of tree?

**Comment by puffpants** [comment]
> Does this only happen with a particular type of tree?

no, it happens to birch, oak, and jungle (haven't tried others). It does randomly work about 5% of the time. I think it's the certain chunks that it works in tho, but I have no like data to back that.
